src/modules/flow/thingspeak/thingspeak.c

```diff
@@ -535,6 +536,7 @@ thingspeak_channel_update_open(struct sol_flow_node *node, void *data, const str
         return -ENOMEM;
     }
 
+    mdata->node = node;
```

Comment:
ok for now, but the best approach is to  remove `mdata->node` and instead of passing `mdata`, give `node` and then query the private data with `sol_flow_node_get_private_data()`.